FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.SchemaManagement.UnitTests/Parquet/FhirParquetSchemaManagerTests.cs

```diff
@@ -31,7 +33,7 @@ static FhirParquetSchemaManagerTests()
         [Theory]
         public static void GivenASchemaType_WhenGetSchema_CorrectResultShouldBeReturned(string schemaType, int propertyCount)
         {
-            var schemaManager = new FhirParquetSchemaManager(_schemaConfigurationOption, NullLogger<FhirParquetSchemaManager>.Instance);
+            var schemaManager = new FhirParquetSchemaManager(_schemaConfigurationOption, _jsonSchemaCollectionsProvider, NullLogger<FhirParquetSchemaManager>.Instance);
```

Comment:
If we would use shcemaManager in every/most tests, can we create it only once before tests?

Reply:
Updated.